bin/node/src/chain_spec.rs

```diff
@@ -253,9 +234,10 @@ pub fn config(
     let faucet_account: AccountId =
         hex!["eaefd9d9b42915bda608154f17bb03e407cbf244318a0499912c2fb1cd879b74"].into();
 
-    let chain_type = match chain_id {
-        DEVNET_ID => ChainType::Development,
-        _ => ChainType::Live,
+    let chain_type = if chain_params.dev_chain {
```

Comment:
this is just my preference, but in almost every case I find pattern matching easier to read than if-else statements (less lines of code if we want to use a metric), but this is by no means me insisting on it

Reply:
In my noobish intuition match is like a heavy duty weapon, whereas if-else is just... good old if-else ;) Btw I'm kinda surprised there's no if-else one-liner in Rust...

Reply:
I'm an animal, I would match even on a boolean value. But I have this killer arg to always show, different project, different languages, but left is a piece of a Solidity contract code soup, right is the same logic replicated in a Lisp using pattern matching on bool arrays :smile:  

![DUZAylGXkAAg1T6](https://user-images.githubusercontent.com/468572/143877915-1290ae7b-1c4e-4f0f-8c6d-131c967abb23.jpeg)

Maybe it's just me but even if someone doesn't fully get the syntax there's a stark difference in readability

Reply:
Ok, I guess you converted me. How do I sign up for Church of Match? ;)